src/Lime.Transport.SignalR/ClientSignalRTransport.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Lime.Protocol;
+using Lime.Protocol.Network;
+using Lime.Protocol.Serialization;
+using Microsoft.AspNetCore.SignalR.Client;
+
+namespace Lime.Transport.SignalR
+{
+    /// <summary>
+    /// Provides methods for client nodes to communicate with a server using SignalR as the underlying transport mechanism.
+    /// This class cannot be inherited.
+    /// </summary>
+    /// <inheritdoc />
+    public sealed class ClientSignalRTransport : SignalRTransport, IDisposable, IAsyncDisposable
+    {
+        private const string FROM_CLIENT_METHOD = "FromClient";
+        private const string HUB_NAME = "envelope";
+        private const string FROM_SERVER_METHOD = "FromServer";
+        private const string CLOSE_METHOD = "Close";
+        private HubConnection _hubConnection;
+        private bool _disposed;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ClientSignalRTransport"/> class.
```

Comment:
Redundant comment - we know that a constructor initiliazes a new instance of the class...

Reply:
@andreminelli  I disagree on this bit though. I think that documentation can only be completely trusted if it is complete. When something is left undocumented, it is ambiguous whether the designer intended to leave it that way because they thought that it is obvious by just looking at the declaration or if they forgot to document it. And then the consumer of the API is forced to test and/or look at the source code in order to know for sure.

There is also precedent for this in the project -- that are many classes that are extensively documented, with the inclusion of what could be called redudant statements. For example, `ChannelBase.Dispose` documents:

> Releases unmanaged and - optionally - managed resources.

`TransportBase` also documents:

> Base class for transport implementation.

And `ChannelInformation.ctor`...

> Initializes a new instance of the <see cref="ChannelInformation"/> class.

Lastly, it would be quite unusual to document only the parameters, leaving the docs without a `summary`. And removing everything would be even worse, not only by following the above logic, but also because we would get warnings from the compiler.

Reply:
These kind of comments are just noise/repetition and should be avoid in my opinion. 
But if you feel more comfortable with them, as you said, there are precedents on this repo indeed...

Reply:
I can certainly change if you feel that we should not encourage this type of docs in new code. It's just that I personally think that writing complete documentation for public APIs, even when they may seem "redudant", is worth it because of the reasons I outlined above. I'm open to discussion though and certainly open to removing it.